src/ui/server/controllers/insurance/your-buyer/working-with-buyer/validation/rules/traded-with-buyer.test.ts

```diff
@@ -0,0 +1,36 @@
+import tradedWithBuyerRule from './traded-with-buyer';
+import FIELD_IDS from '../../../../../../constants/field-ids/insurance/your-buyer';
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import emptyFieldValidation from '../../../../../../shared-validation/empty-field';
+import { RequestBody } from '../../../../../../../types';
+
+const {
+  WORKING_WITH_BUYER: { TRADED_WITH_BUYER: FIELD_ID },
+} = FIELD_IDS;
+
+const {
+  INSURANCE: {
+    YOUR_BUYER: {
+      WORKING_WITH_BUYER: { [FIELD_ID]: ERROR_MESSAGE },
+    },
+  },
+} = ERROR_MESSAGES;
+
+describe('controllers/insurance/your-buyer/working-with-buyer/validation/traded-with-buyer', () => {
```

Comment:
Can we add `rules` to match the file path, probably the same with the other new validation rule